File: VB_Lib/Extensions/ShowPix.vb
Imports System.Drawing

Public Class ShowPix
    WithEvents PicBox As New BetterPictureBox
    Sub New(ByVal Pix As Bitmap, Optional Text As String = "")
        InitializeComponent()
        Me.Text = Text
        If IsNothing(Pix) Then Exit Sub
        PicBox.LoadImage(Pix)
        Me.ShowDialog()
    End Sub
    Sub New(ByVal Pix As Pix, Optional Text As String = "")
        InitializeComponent()
        Me.Text = Text
        If IsNothing(Pix) Then Exit Sub
        PicBox.LoadImage(Pix.ToBitmap)
        Me.ShowDialog()
    End Sub
    Private Sub ShowPix_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        PicBox.Left = 0
        PicBox.Top = 0
        PicBox.Width = Me.Width
        PicBox.Height = Me.Height
        Me.Controls.Add(PicBox)
    End Sub
End Class

File: CS_LIB/Extensions/ShowPix.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace LeptonicaSharp.Extensions
{
    public partial class ShowPix : Form
    {
        BetterPictureBox PicBox = new BetterPictureBox();
        public ShowPix()
        {
            InitializeComponent();
        }

        public ShowPix(Bitmap Pix, string Text = "")
        {
            Load += ShowPix_Load;
            InitializeComponent();
            this.Text = Text;
            if ((Pix == null))
                return;
            PicBox.LoadImage(Pix);
            this.ShowDialog();
        }
        public ShowPix(Pix Pix, string Text = "")
        {
            Load += ShowPix_Load;
            InitializeComponent();
            this.Text = Text;
            if (Pix == null) { return; }
            PicBox.LoadImage(Pix.ToBitmap());
            this.ShowDialog();
        }
        private void ShowPix_Load(object sender, EventArgs e)
        {
            PicBox.Left = 0;
            PicBox.Top = 0;
            PicBox.Width = this.Width;
            PicBox.Height = this.Height;
            this.Controls.Add(PicBox);
        }
    }
}
